specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```
@@ -0,0 +1,355 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Rating And Reviews API",
+    "description": "Microsoft Marketplace Rating And Reviews API",
+    "version": "2023-01-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/products/{uniqueProductId}/reviews": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get reviews by unique product id",
+        "operationId": "Reviews_GetByOfferId",
+        "x-ms-examples": {
+          "Facets_Get": {
+            "$ref": "./examples/GetProductReviews.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "uniqueProductId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id"
+          },
+          {
+            "type": "integer",
+            "name": "maxpagesize",
+            "in": "query",
+            "description": "Maximum number of reviews to return",
+            "format": "int32",
+            "default": 50
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "description": "Version of the REST API used for the request."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a list of reviews for the given unique product",
+            "schema": {
+              "$ref": "#/definitions/PageResultOfReviewDto"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 - In case request malformed.\n\n * 500 - In case internal error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/products/{uniqueProductId}/reviews/{reviewId}/comments": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get the comments of a given review",
+        "operationId": "ReviewComment_GetComments",
```

The verb in the operationId of a get on a collection should be "list".
```suggestion
        "operationId": "ReviewComment_ListComments",
```

---

Done